### PR TITLE
mysql: Debian cleanup

### PR DESCRIPTION
**Pull request by rsalevsky**
Not tested yet. Gating runs with mysql backend.

### CODE REVIEW

**Comment by rsalevsky** [comment]
Tested the change and Gating failed during the Manila setup.

File: chef/cookbooks/mysql/recipes/client.rb [hunk 1]
```diff
@@ -17,10 +17,10 @@
 # limitations under the License.
 #
 
-package "mysql-client" do
+package "mariadb-client" do
   package_name value_for_platform_family(
-    ["rhel", "suse", "fedora"] => "mysql",
-    "default" => "mysql-client"
+    ["rhel", "fedora"] => "mysql",
```
Comment (nicolasbock) [inline chef/cookbooks/mysql/recipes/client.rb]: Why are we removing 'suse' here?

Comment (rsalevsky) [inline chef/cookbooks/mysql/recipes/client.rb]: `suse` is now default ;)

Comment (nicolasbock) [inline chef/cookbooks/mysql/recipes/client.rb]: Well, then :smile:

**Pull request merged**